src/test/java/com/google/sps/ImageSelectionUrlGenTest.java

```diff
@@ -0,0 +1,112 @@
+package com.google.sps.image;
+
+import static org.junit.Assert.assertEquals;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.junit.Test;
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+@RunWith(JUnit4.class)
+public final class ImageSelectionUrlGenTest {
+
+  private final Set<String> keywords = new LinkedHashSet<>();
```

Comment:
There is not really any reason to have this keyword set defined as instance variable.

You may as well define it in every test method. This might add a few lines of code but makes it easier to read each test method without having to jump through the test file. This may be of relatively little concern, but imagine a more complex case where you do this with five different variables from which only some are used by each test case; harder to read.